### PR TITLE
DOC: Add link to logo-repo.

### PR DESCRIPTION
**Pull request by prisae**
This should come together with a README in https://github.com/openjournals/digital-assets/tree/master/joss/logo, explaining how the logo was created and what it means. See openjournals#46

### CODE REVIEW

File: app/views/home/about.html.erb [hunk 1]
```diff
@@ -87,7 +87,7 @@
         <p>Any code snippets included in <%= setting(:abbreviation) %> papers are subject to the <a
             href="https://opensource.org/licenses/MIT" target="_blank">MIT license</a> regardless of the license of the
           submitted software package under review.</p>
-        <p>Any use of the JOSS logo is licensed CC BY 4.0.</p>
+        <p>Any use of the JOSS logo is licensed CC BY 4.0. See the logo directory in the <a rel="https://github.com/openjournals/digital-assets">digital-assets</a>-repository for more information about it.</p>
```
Comment (danielskatz) [inline app/views/home/about.html.erb]: Can you change this to "See the joss/logo directory ..."

**Comment by terrytangyuan** [comment]
Should we also switch to use the new logo for the Github org https://github.com/openjournals?

**Comment by danielskatz** [comment]
@terrytangyuan  - please open a new issue for that, so this one doesn't get too overloaded.
I'll also open a new one for the current sticker.

File: app/views/home/about.html.erb (hunk 1 above)
Comment (prisae) [reply app/views/home/about.html.erb]: Done.

**Comment by danielskatz** [comment]
I don't see this on the page yet - it might have a build cycle we need to wait for, or it might need to wait for @arfon to do something manually.  (Or something else)

**Comment by prisae** [comment]
Thought the same. Thanks @danielskatz !

**Comment by danielskatz** [comment]
This is now on the live page - thanks!

**Pull request merged**